realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -51,6 +52,13 @@
  * is required.
  * <p>
  * A RealmQuery cannot be passed between different threads.
+ * <p>
+ * Queries can be launched from any thread and results are obtained fairly quickly most of the times. However, launching
```

Comment:
This is not true, you can only launch the query from the Thread holding the Realm instance. Same for the other descriptions

Reply:
Yeah, it was a broad generalisation. I will change it.